### PR TITLE
📖 Move Control Plane contracts doc to right chapter

### PR DESCRIPTION
**Pull request by NareshKoduru**
Fixes https://github.com/kubernetes-sigs/cluster-api/issues/5393

/area documentation

### CODE REVIEW

**Comment by NareshKoduru** [comment]
cc @fabriziopandini

File: docs/book/src/SUMMARY.md [hunk 1]
```diff
@@ -102,6 +102,7 @@
           - [Cluster Infrastructure](./developer/providers/cluster-infrastructure.md)
           - [Machine Infrastructure](./developer/providers/machine-infrastructure.md)
           - [Bootstrap](./developer/providers/bootstrap.md)
+          - [Control Plane](./developer/providers/control-plane.md)
```
Comment (fabriziopandini) [inline docs/book/src/SUMMARY.md:105]: nit: I would move this up, below Cluster Infrastructure (L103)

**Review by fabriziopandini — COMMENTED** [review]
Nice Work!, only one nit

File: docs/book/src/SUMMARY.md (hunk 1 above)
Comment (NareshKoduru) [reply docs/book/src/SUMMARY.md]: 👍  , Addressed in [39919b5](https://github.com/kubernetes-sigs/cluster-api/pull/11113/commits/39919b5eb2ade2ab5bef28c55f1a0e16d9854f38). Thanks

**Comment by NareshKoduru** [comment]
@fabriziopandini,
if everything looks good, could you please help with merging the pull request?Thanks

**Comment by fabriziopandini** [comment]
> @fabriziopandini,
> if everything looks good, could you please help with merging the pull request?Thanks

Let's give a few days to other maintainers/reviewers to take a look, there is no rush for this PR

**Comment by NareshKoduru** [comment]
> > @fabriziopandini,
> > if everything looks good, could you please help with merging the pull request?Thanks
> 
> Let's give a few days to other maintainers/reviewers to take a look, there is no rush for this PR

Understood. Thanks

**Comment by sbueringer** [comment]
Thx!

/approve

**Pull request merged**